### PR TITLE
Link to registry page in project description (#992)

### LINKED ISSUE #992: [projects] Link to Verra (or other registry) project details
In the retirement aggregator (and carbon dashboard) we include a link to the associated verra project details page for a project. For [example](https://registry.verra.org/app/projectDetail/VCS/1052).

This appears to be missing for Carbonmark unless it was intended to be a link under Project Documentation? ([figma link](https://www.figma.com/file/5GewbdVaM1Q06CwHmF7d3F/%2Fprojects?node-id=11-58928&t=yi9uQcysBZblHpnW-0))

Image

**Other Notes:**

- Most of the projects are currently Verra but we expect more projects to come into the marketplace from other registries

### PR DESCRIPTION
**Pull request by 0xemc**
## Description

Add a link to `project.url` to the description of a listing

## Related Ticket

Resolves #992

## Changes
image

## Remaining

- [ ] Add additional project document links

### CODE REVIEW

**Comment by jabby09** [comment]
@0xemc @Atmosfearful @0xAurelius 
It looks to me like the URL stored in Sanity is not the expected URL pattern for a verra project so we are getting an error.

Example: 
See this project: https://carbonmark-git-emc-992-link-to-registry-page-klimadao.vercel.app/projects/VCS-1052-2012
VIEW REGISTRY DETAILS from the Sanity DB: https://registry.verra.org/app/projectDetail/VCS/VCS-1052 which gives this error...


The Carbon Dashboard and RA Find Project form use the URL pattern of https://registry.verra.org/app/projectDetail/VCS/1052 which resolves correctly

So it looks like we have an extra VCS- in the URL https://registry.verra.org/app/projectDetail/VCS/**VCS-**1052

File: carbonmark/components/pages/Project/index.tsx [hunk 1]
```diff
@@ -172,12 +175,21 @@ const Page: NextPage<PageProps> = (props) => {
...
+            <div className="description">
+              <Text t="h5" color="lighter">
+                <Trans>Description</Trans>
+              </Text>
+              <Text t="body1">
+                {project.description ?? "No project description found"}
+              </Text>
+            </div>
+            {notNil(project.url) && (
+              <Link href={project.url} className="registryLink">
```
Comment (ladytrekker) [inline carbonmark/components/pages/Project/index.tsx:187]: Link is actually for internal links of the NextJS routing.
For external links there is the `<Anchor>` component. Please compare this again 🙏

**Review by ladytrekker — APPROVED** [review]
Looks good to me.
Please see the suggestion below to use the correct component

**Comment by 0xAurelius** [comment]
> So it looks like we have an extra VCS- in the URL https://registry.verra.org/app/projectDetail/VCS/VCS-1052

Yes this will be fixed soon! Got the code working, just need to upload to Sanity

**Comment by 0xAurelius** [comment]
This should be resolved now - links look correct on my end at the preview URL: 
image

**Comment by jabby09** [comment]
> https://registry.verra.org/app/projectDetail/VCS/1052

@0xAurelius yes agreed, seems to be good now (although the verra page takes FOREVER to load to the point it looks broken)

@Atmosfearful this one I think is good to go if you are done your reviews.

File: carbonmark/components/pages/Project/index.tsx (hunk 1 above)
Comment (Atmosfearful) [inline carbonmark/components/pages/Project/index.tsx:187]: let's open in new tab

**Review by Atmosfearful — CHANGES_REQUESTED** [review]

**Comment by jabby09** [comment]
@0xemc @Atmosfearful shimon picked this up in qa that the implementation of the text does not match the design.
Screenshot 2023-04-20 at 9 16 08 AM

Can you confirm we are using the correct font CSS styles?

Screenshot 2023-04-20 at 9 13 14 AM

File: carbonmark/components/pages/Project/index.tsx (hunk 1 above)
Comment (0xemc) [reply carbonmark/components/pages/Project/index.tsx:187]: Cool, done 👍

Comment (0xemc) [reply carbonmark/components/pages/Project/index.tsx:187]: Good point. Updated.

**Comment by 0xemc** [comment]
> @0xemc @Atmosfearful shimon picked this up in qa that the implementation of the text does not match the design.
> 
> Screenshot 2023-04-20 at 9 16 08 AM
> 
> Can you confirm we are using the correct font CSS styles?
> 
> Screenshot 2023-04-20 at 9 13 14 AM

You're right it was using the primary font (DM Sans) when it should be the secondary (Poppins). Updated

**Review by Atmosfearful — APPROVED** [review]

**Pull request merged**